### ISSUE TITLE
A11y: Take Action & About menu items are not keyboard operable

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#330]
## Describe the task
The items under the Take Action & About menu should be reachable & operable via keyboard. Based on how the `<a>` tag is nested, you can't just press ENTER when it looks like you're on the item to follow the link.

Currently if you TAB to the Take action nav item then press ENTER... The dropdown with page options opens up. Press UP and DOWN arrows to navigate the items. Press ENTER on any item. You should be taken to the link.
<img width="278" alt="Screenshot 2024-02-10 at 12 58 36 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/b6624710-148d-4297-8735-311bcc092700">

Instead, ENTER does nothing. If you press TAB after Up/Down, you will land on the actual link. From there you can press ENTER and follow the link. It also looks weird as well.
<img width="280" alt="Screenshot 2024-02-10 at 12 58 40 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/3c625ad2-3913-48a3-8e43-835859992b61">

## Acceptance Criteria

- [ ] TAB to the Take Action nav item. Press ENTER. The dropdown with page options opens up. Press UP and DOWN arrows to navigate the items. Press ENTER on any item. It should be activated just the same as if you clicked on it.
- [ ] Same for the items in the About menu.

**Comment by bacitracin** [comment]
This has been fixed, so closing
